### DISCUSSION TITLE
Bug reporting

### DISCUSSION THREAD

**Discussion by ArtechFuz3D** [AntiMicroX/antimicrox#299] [Q&A] [answered]
Hey where can I report bugs. When trying more advanced assignments the app often crashes and results in my desktop doing all kinds of strange things even when the application has exited.

**Comment by pktiuk** [comment, marked as answer]
Reporting bugs can be done here https://github.com/AntiMicroX/antimicrox/issues/new?assignees=&labels=bug&template=bug_report.yaml

But before reporting, I recommend [searching for similar issues](https://github.com/AntiMicroX/antimicrox/issues) to avoid duplication.
